README.md

```diff
@@ -497,22 +506,28 @@ Accepts OpenTelemetry Protocol (OTLP) v1.7.0 logs in protobuf format via HTTP. T
 
 The HTTP endpoint accepts `POST` requests with `Content-Type: application/x-protobuf` and stores the decoded logs for retrieval via the `/test/session/logs` endpoint.
 
-### OTLP Logs via GRPC
+### /v1/metrics (HTTP)
+
+Accepts OpenTelemetry Protocol (OTLP) v1.7.0 metrics in protobuf format via HTTP. This endpoint validates and decodes OTLP metrics payloads for testing OpenTelemetry metrics exporters and libraries.
+
+The HTTP endpoint accepts `POST` requests with `Content-Type: application/x-protobuf` and stores the decoded metrics for retrieval via the `/test/session/metrics` endpoint.
```

Comment:
This endpoint (and the logs one) can also accept requests with `Content-Type: application/json` right?

Reply:
Yup and this is tested. I will update the docs